app/views/provider_mailer/chase_provider_decision.text.erb

```diff
@@ -0,0 +1,15 @@
+Dear <%= @application.provider_user_name || 'colleague' %>
+
+# Only 20 working days left to respond
```

Comment:
Similar to https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1290#discussion_r375850392, we probably don't want this number hard-coded

Reply:
That makes sense, resolved in 82b6bef